packages/vaadin-crud/test/crud.test.js

```diff
@@ -864,6 +864,25 @@ describe('crud', () => {
       expect(crud.$.toolbar.style.display).to.be.not.equal('none');
     });
 
+    it('should have hide-toolbar attribute when property set to true', () => {
+      crud.hideToolbar = true;
+      expect(crud.hasAttribute('hide-toolbar')).to.be.equal(true);
+    });
+
+    it('should remove hide-toolbar attribute when set to false on propery', async () => {
+      crud.hideToolbar = true;
+      await nextRender(crud);
```

Comment:
This is not needed: in Polymer property changes are synchronous. so the attribute will be set right away.